presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestPrestoDriverAuth.java

```diff
@@ -0,0 +1,163 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.jdbc;
+
+import com.facebook.presto.server.testing.TestingPrestoServer;
+import com.facebook.presto.sql.parser.SqlParserOptions;
+import com.facebook.presto.tpch.TpchPlugin;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logging;
+import io.airlift.units.Duration;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Base64;
+import java.util.Map;
+import java.util.Properties;
+
+import static com.google.common.io.Resources.getResource;
+import static io.airlift.testing.Assertions.assertLessThan;
+import static io.airlift.units.Duration.nanosSince;
+import static java.lang.String.format;
+import static java.nio.charset.StandardCharsets.US_ASCII;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+public class TestPrestoDriverAuth
+{
+    private static final String TEST_CATALOG = "test_catalog";
+    private static final String JWT_KEY = Base64.getEncoder().encodeToString("jwt-signing-key".getBytes(US_ASCII));
+    private TestingPrestoServer server;
+
+    @BeforeClass
+    public void setup()
+            throws Exception
+    {
+        Logging.initialize();
+
+        server = new TestingPrestoServer(
+                true,
+                ImmutableMap.<String, String>builder()
+                        .put("http-server.authentication.type", "JWT")
+                        .put("http.authentication.jwt.key", JWT_KEY)
+                        .put("http-server.https.enabled", "true")
+                        .put("http-server.https.keystore.path", getResource("localhost.keystore").getPath())
+                        .put("http-server.https.keystore.key", "changeit")
+                        .build(),
+                null,
+                null,
+                new SqlParserOptions(),
+                ImmutableList.of());
+        server.installPlugin(new TpchPlugin());
+        server.createCatalog(TEST_CATALOG, "tpch");
+        waitForNodeRefresh(server);
+    }
+
+    private static void waitForNodeRefresh(TestingPrestoServer server)
```

Comment:
Is this needed? We do a force refresh in `TestingPrestoServer`. I don't know why this is in `TestPrestoDriver`

Reply:
If we do need this, import the one from `TestPrestoDriver`